DiamondLightSource/python-artemis
# 391 ispyb interaction and comment

Fixes #391 #393 
Incidentally fixes #390 
Includes changes from #400
Loads existing comment from ispyb and appends if there was an error

### To test:
1. Run tests
2. Ideally, run a grid scan with S03, and the dev ispyb config in the $ISPYB_CONFIG_PATH env variable, inspect the logs and see that the comment is loaded
